jax/numpy/lax_numpy.py

```diff
@@ -3060,6 +3060,75 @@ def replace(tup, val):
 def take_along_axis(arr, indices, axis):
   return _take_along_axis(arr, indices, axis)
 
+
+### SetOps
+
+def _unique1d_sorted_mask(ar, optional_indices=False):
```

Comment:
If this is jittable, you might consider wrapping it in `jit` here, as that would likely speed up `np.unique` overall.

Reply:
Thank you for the comment! I added a jit wrapper to the function.